hex/helpers/build

```diff
@@ -10,7 +10,14 @@ fi
 install_dir="$DEPENDABOT_NATIVE_HELPERS_PATH/hex"
 mkdir -p "$install_dir"
 
-mix local.hex --force
```

Comment:
This is the second time we're calling `mix local.hex --force` on `main`, so need to either remove it from here or remove it from the Dockerfile... that's why I opted to centralize the logic in here.